lib/std/os/bits/openbsd.zig

```diff
@@ -789,11 +789,22 @@ pub const _ksiginfo = extern struct {
             addr: ?*c_void,
             trapno: c_int,
         },
+        __pad: [128 - 3 * @sizeOf(c_int)]u8,
```

Comment:
I found it is error prone. the size is right (`116 u8`), but the definition in OpenBSD is `29 c_int`. it will be complex to track it if it changes at some point.